vmware-event-router/internal/integration/openfaas_test.go

```diff
@@ -68,29 +70,22 @@ var _ = Describe("OpenFaaS Processor", func() {
 			It("should not receive a failed invocation response", func() {
 				Expect(failCount).To(Equal(0))
 			})
-
-			It("should not error", func() {
```

Comment:
I suppose this removal is since we moved the no error expectation above?

Reply:
Yes, I moved it closer to the actual line above, reducing one additional assertion (and time).

Reply:
Yep, it's better that way